e2e_testing/scripts/apply-fixtures.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+directory="e2e_testing/fixtures"
+
+for path in "$directory"/*; do
+	if [ -f "$path" ]; then
+		file=$(basename $path)
+		echo "Applying: $file"
+		docker compose -f docker-compose-e2e-tests.yml cp $path db:/$file
+		docker compose -f docker-compose-e2e-tests.yml exec -u postgres db psql postgres postgres -f /$file
```

Comment:
If I run locally, it fails due to some of this data already existing.

It might be better to insert this data into a separate e2e-only database that is recreated on each run, if possible, so that it doesn't alter the main database.  For example the python unit tests run on a separate `test_postgres` database.

Reply:
> If I run locally, it fails due to some of this data already existing.
> 
> It might be better to insert this data into a separate e2e-only database that is recreated on each run, if possible, so that it doesn't alter the main database. For example the python unit tests run on a separate `test_postgres` database.

I've added a named volume, `pgdata-e2e-tests` to the Docker Compose file so the tests will use their own database instance.

Reply:
Now applying via the manage.py tool, which doesn't have this issue.